README.md

```diff
@@ -565,3 +565,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 ### Programming Courses Free Online
 
 - [SoloLearn](https://www.sololearn.com/learning)
+
+### VS Code tricks
```

Comment:
Can you change the title to just "VS Code"?

Reply:
Ok ,I just changed it.